mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
```

Comment:
Remove this file, it is not related to PR

Reply:
It's already removed

Reply:
Do you mean that you want to remove it from the git history? Or it's okay to have the final commit without it?